services/telemetry/telemetry.go

```diff
@@ -696,6 +697,7 @@ func (ts *TelemetryService) trackConfig() {
 	})
 
 	ts.sendTelemetry(TrackConfigNativeApp, map[string]interface{}{
+		"isdefault_app_custom_url_schemes":    reflect.DeepEqual(cfg.NativeAppSettings.AppCustomURLSchemes, model.GetDefaultAppCustomURLSchemes()),
```

Comment:
Couple of things here:
- Do we care about the order of elements in the slice? `reflect.DeepEqual` doesn't.
- I'd suggest to include this functionality in `isDefault` itself. We can just check for the type, and then check the 2 slices. Importing the reflect package seems too heavy for this.

Reply:
Agreed.

Reply:
Thanks! I have added the equality check in `isDefault`. I don't think we should cover the order of elements in the slice as it could mean value has been set something other than default.